api/kyverno/v1/image_verification_types.go

```diff
@@ -151,10 +151,10 @@ type StaticKeyAttestor struct {
 }
 
 type SecretReference struct {
-	// name of the secret
+	// Name of the secret. The provided secret must contain a key named cosign.pub.
 	Name string `json:"name" yaml:"name"`
 
-	// namespace name in which secret is created
+	// Namespace name in which secret is created
```

Comment:
"Namespace name where the Secret exists."

Reply:
fixed